### PR TITLE
Database entity framework migrations

### PR DESCRIPTION
**Pull request by SamJDriver**
I think I'm calling it good on the migrations for now. I set up some more secrets and secured more stuff for the database. I cleaned up the program.cs file and experimented around with some scripts to do the release. I think we might just not need them though, so I'm not gonna spend too much more time on it.

### CODE REVIEW

File: GameReviewBackend/API/Program.cs [hunk 1]
```diff
@@ -167,7 +164,15 @@ public static void Main(string[] args)
             app.UseAuthentication();
             app.UseAuthorization();
 
-            app.UseMiddleware<GlobalExceptionHandlingMiddleware>();
+            if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == "Development")
```
Comment (nisule) [inline GameReviewBackend/API/Program.cs]: minor but I think I would switch this logic around so if its == "production" then use prod

File: GameReviewBackend/BusinessLogic/Infrastructure/IgdbApiService.cs [hunk 2]
```diff
@@ -15,8 +15,9 @@ namespace BusinessLogic.Infrastructure
     public class IgdbApiService : IIgdbApiService
     {
         GenericRepository<DockerDbContext> _genericRepository;
-        private string CLIENT_ID = "0w2j0mktoxkvehox8hl42p15tladu0";
-        private string CLIENT_SECRET = "gpptkmravp51lkhl5d22msu8i2rs2f";
+        private readonly string CLIENT_ID = "0w2j0mktoxkvehox8hl42p15tladu0";
+        private readonly string CLIENT_SECRET = "gpptkmravp51lkhl5d22msu8i2rs2f";
```
Comment (nisule) [inline GameReviewBackend/BusinessLogic/Infrastructure/IgdbApiService.cs]: this secret should probably not be hardcoded

File: GameReviewBackend/API/Program.cs (hunk 1 above)
Comment (SamJDriver) [reply GameReviewBackend/API/Program.cs]: Switched these around

**Review by nisule — APPROVED** [review]
looks good

File: GameReviewBackend/BusinessLogic/Infrastructure/IgdbApiService.cs (hunk 2 above)
Comment (SamJDriver) [reply GameReviewBackend/BusinessLogic/Infrastructure/IgdbApiService.cs]: removed these and replaced with config secrets

**Pull request merged**